lib/fastlane/plugin/wpmreleasetoolkit/actions/android/an_update_metadata_source_action.rb

```diff
@@ -71,17 +71,17 @@ def self.create_block_parsers(release_version, block_files)
         @blocks = Array.new
 
         # Inits default handler
-        @blocks.push (Fastlane::Helper::UnknownMetadataBlock.new)
+        @blocks.push Fastlane::Helper::UnknownMetadataBlock.new
```

Comment:
I'm not sure I understand this one – `push` is a function, so AFAICT shouldn't the brackets be allowed? 

I don't object to it – I just don't understand 😅

Reply:
Yeah that one surprised me as well when the linter warned about it.

The thing is, we could have kept the parentheses, but if we did, we would have needed to remove the spaces between `push` and the open bracket.

```ruby
# Here it's interpreted as a function call on which we just omit the parentheses around parameters, OK
@blocks.push Fastlane::Helper::UnknownMetadataBlock.new

# Here it's interpreted as a function call on which we explicitly use the parentheses around parameters, still OK
@blocks.push(Fastlane::Helper::UnknownMetadataBlock.new)

# But here it's interpreted as a function call on which we omit the parentheses around parameters…
# and then pass a single parameter to it that happens to be wrapped in parentheses, i.e. interpreted as a sub-expression
@blocks.push (Fastlane::Helper::UnknownMetadataBlock.new)
```